docs/commands/browser/assertView.mdx

```diff
@@ -24,7 +25,7 @@ await browser.assertView(state, selector, options);
 </thead>
 <tbody>
 <tr><td>[state](#state)</td><td>String</td><td>Обязательный параметр. Название состояния теста. Должно быть уникальным в пределах одного теста.</td></tr>
-<tr><td>[selector](#selector)</td><td>String или String[]</td><td>Обязательный параметр. Селектор DOM-элемента, который необходимо заснять.</td></tr>
+<tr><td>[selector](#selector)</td><td>String или String[]</td><td>Необязательный параметр. Может быть пропущен. Селектор DOM-элемента, который необходимо заснять.</td></tr>
```

Comment:
Let's leave a note right here that if omitted, the whole viewport will be screenshotted